listenbrainz/db/dump.py

```diff
@@ -121,6 +121,7 @@
         'latest_import',
         'musicbrainz_row_id',
         'gdpr_agreed',
+        'email',
```

Comment:
I think we also need to add an empty '' for email like we do for auth_token to https://github.com/metabrainz/listenbrainz-server/blob/master/listenbrainz/db/dump.py#L58-L68